docs/fastdds/library_overview/includes/exec_model.rst

```diff
@@ -102,6 +124,9 @@ but the default configuration is to always use a metatraffic port and a unicast
 This applies both to UDP and Shared Memory since TCP does not support multicast.
 More information can be found at the :ref:`listening_locators_default` page.
 
+*Fast DDS* offers the possibility of configuring certain attributes of the threads it creates by means of the
+:ref:`threadsettingsqos`.
+
```

Comment:
Apart from this line, should we add a cross-reference to the QoS where the thread can be configured?
We could do it in the description column, with something like `Can be configured through xxxQos`